docs/manual/mustcall-checker.tex

```diff
@@ -0,0 +1,164 @@
+\htmlhr
+\chapterAndLabel{Must Call Checker}{mustcall-checker}
+
+The MustCall Checker tracks the methods that an object should call before it is deallocated
+and produces a conservative overapproximation of the set of methods that might
```

Comment:
What is the relationship between the clauses connected by this "and"?
What is the relationship between "an object should call" and "might actually need to be called on an object"?